app/component/itinerary/ItineraryPage.js

```diff
@@ -745,16 +739,63 @@ export default function ItineraryPage(props, context) {
     );
   }
 
+  /**
+   * Checks if itinerary in local storage was set with identical parameters.
+   *
+   * @param {{from: string, to: string, time: number, arriveBy: boolean, hash: number}} storedParams
+   * @param {boolean} shouldCompareHash false bypasses hash check when user navigates to ItineraryPage root with the list of suggested itineraries
+   * @returns true if stored itinerary was set with identical parameters
+   */
+  const isMatchingParams = (storedParams, shouldCompareHash) => {
```

Comment:
I think these should be moved in some util file if possible, in order not to bloat the ItineraryPage more than necessary.